### ISSUE TITLE
Species details: truncation of text

### ISSUE THREAD

**Issue by reupost** [rukayaj/ssp#30]
Some text is truncated, e.g. for Past Justification for Encephalartos eugene-maraisii I.Verd.  Is it possible to repair this to allow the taxon experts to see the full notes that were captured for the 2010 assessment?

**Comment by reupost** [comment]
Could you chase Brenda up on this, or shall I ask her?  I'd take a look on Kimserver if that still exists, it might be there.

**Comment by rukayaj** [comment]
I did have a quick squizz around and didn't see anything, I'll have another closer look. And yes I did chase her, but you know how elusive she can be...

**Comment by reupost** [comment]
I've looked at the original 2010 spreadsheets e.g. 'SensitiveTaxa_Plants_merge_12May2010.xlsx' with tabs like 'InclusionList_29042010' and the truncation is present there as well.  I suspect it is text that came from the TSP database at that time.  Looking at recent copies of the TSP database has not helped, since these taxa and fields have changed in the interim.

Given that the plant assessment is going to only be at the genus level at this stage, I suggest we deprecate this issue.

**Comment by rukayaj** [comment]
Let's close this then.

**Comment by reupost** [comment]
I've tracked down the details in a 2009 backup of the TSP database. Here's the content of the email that described what was done:
Ok, I have created a new spreadsheet for the plant folk to use (only Tania at present).  This contains the 226* species on the public plant list, but I have added the additional details originally found on the big list of species that was being made available for download, as well as fixing the truncation of some of the threat information.  I have not included all the supplementary species though, its just the 226 species.

I also haven’t dug through the 2009 RedList database to extract the population and growth form information for the additional 59 species, nor spent too long on getting the justifications from the original 2010 sensitive species spreadsheets (those I could match easily from the ‘Inclusion’ tab I did, but this was only for 9 species).

As Rukaya mentioned, the missing species may have been due to taxonomic changes since 2010.  For the purposes of things clean I have kept the names as per the 2010 list.  If Tania et al need more current information then we can always extract that specifically for them, but it doesn’t seem worth putting in a lot of time updating this list with the latest taxonomy, status, etc.

Other changes:
1.	Species number drops from 226 to 224,  since Aloe brevifolia and Drimia cooperi were each included twice in the 2010 list, with different authorities.  I’ve consolidated these into single species entries.
2.	I’ve fixed species names so that they are all complete with authorities; previously there was a mix of full and naked names.  I also removed any “[1]”-style BRAHMS tags from the names.
3.	Removed the ‘include’ column, since this seemed confusing
